.travis.yml

```diff
@@ -20,14 +20,17 @@ perl:
 before_install:
     - sudo apt-get install -y libidn11-dev
     - eval $(curl https://travis-perl.github.io/init) --auto
-    - local-lib
+
+    - sudo apt-get install -y libidn11-dev
```

Comment:
Should that really be installed a second time?

Reply:
They should not. I'll remove one of the commands.